### PR TITLE
35 Ruuter service to create resql services

### PR DESCRIPTION
**Pull request by baha-a**
Related to [Issue #35](https://github.com/buerokratt/Service-Module/issues/35),

According to [this comment](https://github.com/buerokratt/Service-Module/issues/35#issuecomment-1447674981) the functionality to list/create/delete files should be part of the Training Module, however this functionality is not ready yet, So I created a simple nodejs service to handle this case, later we can remove it and use the Training Module's.

### CODE REVIEW

File: DSL/Node/src/file.ts [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import express, { Router } from 'express';
+import path from "path";
+import fs from "fs";
```
Comment (trevorling) [inline DSL/Node/src/file.ts:3]: Let's use the single quotes for the imports.
Please look over all the places- it's for consistency.

File: docker-compose.yml [hunk 2]
```diff
@@ -50,16 +50,29 @@ services:
...
       - ./DSL/DMapper/js:/workspace/app/js/services
-     ports:
+    ports:
       - 3000:3000
-     networks:
-      - bykstack 
+    networks:
+      - bykstack
+  custome_node_server:
+    container_name: custome_node_server
```
Comment (trevorling) [inline docker-compose.yml:64]: Lines 63 & 64:
`custome`_node_server -> `custom`_node_server

**Review by trevorling — CHANGES_REQUESTED** [review]

File: DSL/Node/src/file.ts (hunk 1 above)
Comment (baha-a) [reply DSL/Node/src/file.ts]: Done.

File: docker-compose.yml (hunk 2 above)
Comment (baha-a) [reply docker-compose.yml]: Fixed.

**Comment by baha-a** [comment]
> @baha-a Sorry, 1 thing that I didn't noticed- indent 4 -> 2

@trevorling, Fixed.

**Review by trevorling — APPROVED** [review]

**Pull request merged**